heartofthealien/heartofthealien-1.2.2.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of the Alien 1.2.2 game port for BeOS"
+HOMEPAGE="http://hota.sourceforge.net"
+COPYRIGHT="2004 Gil Megidish"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="git://github.com/ytmytm/beos-heartofthealien"
+SOURCE_DIR="heartofthealien"
+PATCHES="heartofthealien-1.2.2.patch"
```

Comment:
Please make these neutral changes, so that the recipe can include a SHA256SUM:
```diff
 REVISION="1"
+srcGitRev="8f1f525fcf50c4610302e870c85625c8ea2f33eb"
-SOURCE_URI="git://github.com/ytmytm/beos-heartofthealien"
+SOURCE_URI="https://github.com/ytmytm/beos-heartofthealien/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1f5d382c283a94a2c743746d7d3340671dfd965c4394fe66a86fcccab759dfb2"
+SOURCE_FILENAME="heartofthealien-$portVersion~$srcGitRev.tar.gz"
-SOURCE_DIR="heartofthealien"
+SOURCE_DIR="beos-heartofthealien-$srcGitRev"
-PATCHES="heartofthealien-1.2.2.patch"
+PATCHES="heartofthealien-$portVersion.patch"
```